pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1520,6 +1523,9 @@ type S3StorageProvider struct {
 	// Path is the full path where the backup is saved.
 	// The format of the path must be: "<bucket-name>/<path-to-backup-file>"
 	Path string `json:"path,omitempty"`
+	// LogBackupPath is the full path where the log backup is saved.
+	// The format of the path must be: "<bucket-name>/<path-to-backup-file>"
+	LogBackupPath string `json:"logBackupPath,omitempty"`
```

Comment:
Can you use `Path` directly?

Reply:
`Path` is the snapshot backup path, it is usually a periodic backup, which is multiple and need use `id` to distinguish, but log backup is usually only one takes effect, so they may be different.
for example, snapshot backup is `xxxx/backup/id_1`, log backup is `xxxx/logbackup`.